src/tally.cpp

```diff
@@ -59,17 +58,44 @@ pyne::Tally::Tally(std::string type, std::string part_name,
   normalization = norm;
 }
 
-// Destructor
-pyne::Tally::~Tally() {
+pyne::Tally::Tally(std::string part_name, std::string ent_geom, double orgn[3],
+                   std::vector<double> mesh_i, std::vector<double> mesh_j, std::vector<double> mesh_k,
+                   std::vector<int> i_ints, std::vector<int> j_ints, std::vector<int> k_ints, 
+                   std::vector<double> e, std::vector<int> e_ints, 
+                   std::string tal_name, double norm, double vec_[3], double axl_[3]) {
+  // Empty Tally Constructor
+  entity_type = "Mesh";
+  entity_name = "";
+  particle_name = pyne::particle::name(part_name);
+  entity_geometry = ent_geom;
+  tally_name = tal_name;
+  entity_size = -1;
+  
+  for (int i=0; i<3; i++)  {
+    origin[i] = orgn[i];
+    vec[i] = vec_[i];
+    axl[i] = axl_[i];
+  }
+  i_meshs = mesh_i;
```

Comment:
why not store these in a vector - you end up using them that way eventually

Reply:
I am not sure which one your talking, the `X_meshs` are already vectors, are you talking about `vec` and `axl`

Reply:
I mean a vector of vectors.  Later on, you do that to loop through the different dimensions, so I thought it would make sense to do that now

Reply:
in line 495, you make an array:
```
std::vector<double> meshes[3] = {i_meshs, j_meshs, k_meshs};
```
It is the only place that you use `i_meshs`, etc, so why not declare them initially to just be elements of `meshes[]`